### ISSUE TITLE
Duplicate icons in macOS

### ISSUE THREAD

**Issue by lobodpav** [bartosz-antosik/vscode-spellright#475]
Issue Type: <b>Bug</b>

1. Start VSCode
2. Click and hold Cmd+Tab
3. The VSCode icon can be seen twice

Extension version: 3.0.60
VS Code version: Code 1.63.2 (899d46d82c4c95423fb7e10e68eba52050e30ba3, 2021-12-15T09:37:28.172Z)
OS version: Darwin x64 21.2.0
Restricted Mode: No

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Core(TM) i7-9750H CPU @ 2.60GHz (12 x 2600)|
|GPU Status|2d_canvas: enabled<br>gpu_compositing: enabled<br>metal: disabled_off<br>multiple_raster_threads: enabled_on<br>oop_rasterization: enabled<br>opengl: enabled_on<br>rasterization: enabled<br>skia_renderer: disabled_off_ok<br>video_decode: enabled<br>webgl: enabled<br>webgl2: enabled|
|Load (avg)|2, 2, 2|
|Memory (System)|16.00GB (0.16GB free)|
|Process Argv|--crash-reporter-id f33747da-e164-4294-9374-eeb7481bd95a|
|Screen Reader|no|
|VM|0%|
</details><details>
<summary>A/B Experiments</summary>

```
vsliv368:30146709
vsreu685:30147344
python383cf:30185419
vspor879:30202332
vspor708:30202333
vspor363:30204092
pythontb:30283811
pythonptprofiler:30281270
vshan820:30294714
vstes263:30335439
vscorecescf:30384386
pythondataviewer:30285071
vscod805cf:30301675
pythonvspyt200:30340761
binariesv615:30325510
bridge0708:30335490
bridge0723:30353136
vsaa593:30376534
vsc1dsc:30421290
pythonvs932:30410667
vscop804:30404766
vs360:30404995
vsrem710cf:30416617
vsccppwtstudy:30421978

```

</details>

**Comment by Tokazama** [comment]
I've also been getting this and it only goes away when removing this extension

**Comment by DominicGBauer** [comment]
Yeah this is also mentioned here https://github.com/microsoft/vscode/issues/138712
